03.bowling/bowling.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+FRAMES = 10
+
+def generate_frames
+  pins_text = ARGV[0]
+  pins = pins_text.split(',')
+  frames = []
+  pins.each do |pin|
+    if pin == 'X'
+      frames << 10
+      frames << 0
+    else
+      frames << pin.to_i
+    end
+  end
+  frames.each_slice(2).to_a
+end
+
+def sum_score(frames)
```

Comment:
sum（合計する）でもいいのですが、単純な合計ではないので、calc（計算する）とした方が、メソッド内に何かしらのロジックがあることをイメージしやすくなるかもしれません。（微妙な違いですけどね）

```suggestion
def calc_score(frames)
```